packages/e2e-tests/src/specs/wordpress/webStoriesBlock.js

```diff
@@ -34,8 +34,8 @@ import {
 import { addAllowedErrorMessage } from '../../config/bootstrap';
 
 const EMBED_BLOCK_CONTENT = `
-<!-- wp:web-stories/embed {"url":"https://preview.amp.dev/documentation/examples/introduction/stories_in_amp","title":"Stories in AMP - Hello World","poster":"https://amp.dev/static/samples/img/story_dog2_portrait.jpg"} -->
-<div class="wp-block-web-stories-embed alignnone"><amp-story-player style="width:360px;height:600px" data-testid="amp-story-player"><a href="https://preview.amp.dev/documentation/examples/introduction/stories_in_amp" style="--story-player-poster:url('https://amp.dev/static/samples/img/story_dog2_portrait.jpg')">Stories in AMP - Hello World</a></amp-story-player></div>
+<!-- wp:web-stories/embed {"url":"https://wp.stories.google/stories/intro-to-web-stories-storytime","title":"Stories in AMP - Hello World","poster":"https://amp.dev/static/samples/img/story_dog2_portrait.jpg"} -->
```

Comment:
Theoretically the title and poster should be updated accordingly too for consistency, but that doesn't really affect the tests, so not a big deal.